### PR TITLE
App 1306 - Success message improvements

### PR DESCRIPTION
**Pull request by luanapp**

### CODE REVIEW

File: src/main/webapp/templates/commentCreatedDialog.hbs [hunk 1]
```diff
@@ -1 +1,4 @@
-<h1>Comment created</h1>
\ No newline at end of file
+<div class="footer">
+    <img class="entity-icon icon" src="{{successImg}}" />
```
Comment (robson-sanchez) [inline src/main/webapp/templates/commentCreatedDialog.hbs]: Is this css class required?

File: src/main/webapp/templates/userAssignedDialog.hbs [hunk 2]
```diff
@@ -1 +1,2 @@
-<h1>Issue {{issueKey}} assigned to {{prettyName}}</h1>
\ No newline at end of file
+<select:user id="user"/><br/>
+<img class="entity-icon icon" src="{{successImg}}" />
```
Comment (robson-sanchez) [inline src/main/webapp/templates/userAssignedDialog.hbs]: The same comment above

File: src/main/webapp/templates/userAssignedDialog.hbs [hunk 3]
```diff
@@ -1 +1,2 @@
-<h1>Issue {{issueKey}} assigned to {{prettyName}}</h1>
\ No newline at end of file
+<select:user id="user"/><br/>
```
Comment (robson-sanchez) [inline src/main/webapp/templates/userAssignedDialog.hbs]: Are you assuming the select primitive will show the selected user?

**Review by robson-sanchez — CHANGES_REQUESTED** [review]

File: src/main/webapp/templates/userAssignedDialog.hbs (hunk 3 above)
Comment (luanapp) [reply src/main/webapp/templates/userAssignedDialog.hbs]: The current behaviour do that, despite being wrong. This fix is addressed in another bug (APP-1314)

Comment (robson-sanchez) [reply src/main/webapp/templates/userAssignedDialog.hbs]: I think we should remove this component for while. Once the bug is fixed we can put this again.

**Review by robson-sanchez — CHANGES_REQUESTED** [review]

**Review by robson-sanchez — APPROVED** [review]

**Pull request merged**